README.md

```diff
@@ -37,34 +41,48 @@ Choose "Add Other" and browse to where you generated the file. You may need to c
 <img src="images/confirm.png" width="600" alt="Confirm">
 Confirm your selection and <strong>do not</strong> check the box to copy the file.
 
-> [!IMPORTANT]
-> The generated Linkmap.txt file must be included in your UI test target. You don't need to specificly use `"$(PROJECT_DIR)/Linkmap.txt"` as the path, only that whatever the `LD_MAP_FILE_PATH` is set is also the file that you include in Copy Bundle Resources.
+> [!NOTE]
+> You don't need to specifically use `"$(PROJECT_DIR)/Linkmap.txt"` as the path. Whatever value is set in `LD_MAP_FILE_PATH` is also the file that you include in Copy Bundle Resources.
```

Comment:
This didn't actually seem super important, and we already call out in a similar note in its section that this isn't actually a requirement.